app/views/admin/categories/edit.html.erb

```diff
@@ -7,9 +7,9 @@
 
 <%= locale_select if AppSettings['i18n.available_locales'].count > 1 %>
 
-<%= bootstrap_form_for @category, url: admin_category_path(id: @category.id), label_errors: true, method: 'patch' do |f| %>
+<%= simple_form_for @category, validate: true, url: admin_category_path(id: @category.id), label_errors: true, method: 'patch' do |f| %>
   <% Globalize.with_locale(params['lang']) do %>
-    <%= f.alert_message "Please fix the errors below." %>
+    <%= render 'shared/error_messages', object: f.object %>
```

Comment:
This is not a blocker but I didn't see this actually getting displayed- I think the client side validations just prevent submit if there are errors.  I guess this is in case JS is not enabled?